test/cookbooks/test/recipes/default.rb

```diff
@@ -47,7 +47,7 @@
   only_if { false }
 end
 
-execute 'uname' do
+execute 'whoami' do
   weight 2
 end
```

Comment:
It's absolutely not related to the change, right ?

Reply:
it actually is, tests were not passing on windows otherwise.
But knowing why we need to do that now is a mystery at the moment